metainfo.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>io.github.andrewmkiss.Demeter</id>
+  
+  <name>Demeter</name>
+  <summary>Process X-ray Absorption Spectroscopy Data</summary>
+  <developer_name>Andrew M. Kiss (flatpak)</developer_name>  <!-- Maintainer -->
```

Comment:
this should be the name of the developers of the application. It can be just "The $APPLICATION contributors" when there are many contributors (of course replace with the application name)